testutils/network/network.go

```diff
@@ -25,6 +25,10 @@ import (
 type testNetwork struct {
```

Comment:
remove testNetwork and replace it with mock

Reply:
let's do it in segregated pr